README.md

```diff
@@ -7,3 +7,11 @@ a function.
 
 Functionary (codename) is the next generation of BEERGARDEN. It is currently in
 the rapid prototyping phase of development. Lots more info coming soon.
+
+## Contributing
+
+See the following guides to get started working on functionary:
+
+- [functionary](./functionary/README.md) - The main application, built on django
+- [runner](./runner/README.md) - Runners handle actual function execution
+- [VSCode](./VSCODE.md) - A guide to using VSCode for functionary development
```

Comment:
The problem with moving this file, is that now it's not available from the functionary.code-workspace. You can open a separate file but it's just a floating tab. Do we need the start of a docs folder too?

Reply:
I don't know that I 100% agree that what you're describing is a problem, but having a dedicated docs directory seems inevitable, so I added one and put the VSCODE.md in there.